checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py

```diff
@@ -97,7 +100,12 @@ def _try_evaluate(input_str: Union[str, bool]) -> Any:
                     return json.loads(input_str)
                 return input_str
             except Exception:
-                return input_str
+                try:
+                    # Remove trailing commas before } or ]
+                    input_str_no_trailing = re.sub(r',(\s*[}\]])', r'\1', input_str)  # type:ignore[arg-type]
```

Comment:
why is this change needed? I believe this might be also a big part of the time difference in performance, as regex operations are long